Source/CombatExtended/CombatExtended/Jobs/Utils/JobGiverUtils_Reload.cs

```diff
@@ -130,7 +130,7 @@ private static Thing FindBestAmmo(Pawn pawn, Building_Turret turret)
 			var ammoComp = turret.GetAmmo();
 			AmmoDef requestedAmmo = ammoComp.SelectedAmmo;	
 			var bestAmmo = FindBestAmmo(pawn, requestedAmmo);   // try to find currently selected ammo first
-			if (bestAmmo == null && ammoComp.EmptyMagazine && requestedAmmo.AmmoSetDefs != null && turret.Faction == Faction.OfPlayer)
+			if (bestAmmo == null && ammoComp.EmptyMagazine && requestedAmmo.AmmoSetDefs != null && turret.Faction != Faction.OfPlayer)
```

Comment:
How does this affect your own mortars' reloading?

Reply:
From testing,  raiders will automatically change ammo types if they run out, but colonists will not. I consider this to be the desired behavior.

Reply:
Player faction turrets don't automatically change ammo types, as was the plan in 7a0a7cb91d6e0b7ca77c91d2b8239b1cf10de0c2.